python-main.js

```diff
@@ -353,6 +353,7 @@ function web_editor(config) {
     // Loads Python code into the editor and filesystem main.py, keeps the rest of files
     function loadPy(filename, codeStr) {
         var isModule = isPyModule(codeStr);
+        var filesystemName = filename;
```

Comment:
This change and the one in line 370 (`setName`) should go in a different PR against the `filesystem` branch instead of `beta-integration`.

The other changes are fine here, as they were caused by to a merge conflict resolution in this branch.

Reply:
See #124

Reply:
Yous still need to remove the `filesystemName` changes from this PR.